build.gradle

```diff
@@ -29,4 +29,10 @@ configure(subprojects - project("arrow-meta")) {
     dependencies {
         signature 'net.sf.androidscents.signature:android-api-level-21:5.0.1_r2@signature'
     }
+
+    tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
+        kotlinOptions {
+            jvmTarget = "1.8"
+        }
+    }
```

Comment:
Awesome job in this PR :clap: Please, could you locate these lines in `arrow` repository? `subprojects` section in `generic-conf.gradle`. `JAVA_VERSION` is available. Thanks!!